packages/esm-bed-management-app/src/bed-administration/bed-administration-table.scss

```diff
@@ -20,12 +20,29 @@
   }
```

Comment:
Can we remove the duplicate `widgetCard` style declarations on line 4 through 21?